modules/gltf/src/gltf/create-gltf-model.ts

```diff
@@ -101,14 +101,14 @@ export function createGLTFModel(device: Device, options: CreateGLTFModelOptions)
     topology: geometry.topology,
     vertexCount,
     modules: [pbr],
-    defines: parsedMaterial.defines,
-    // parameters: parsedMaterial.parameters,
-    parameters, // TODO use/merge parsedMaterial
     vs: addVersionToShader(device, vs),
     fs: addVersionToShader(device, fs),
-    bindings: parsedMaterial.bindings,
-    uniforms: parsedMaterial.uniforms,
-    ...modelOptions
+    ...modelOptions,
+
+    bindings: {...parsedMaterial.bindings, ...modelOptions.bindings},
```

Comment:
Ugh, this is a lot for devs to keep track of. We really need the `ShaderInputs` class I am working on :)